tests/integration/commands/generateKey.spec.ts

```diff
@@ -0,0 +1,77 @@
+// import { parseJwk } from 'jose/jwk/parse'
+import { EOL } from 'os';
+import { readFile } from 'fs/promises';
+import { executeCli } from '../../helpers/execute';
+
+describe('generate-key', function () {
+  describe('Happy Path', function () {
+    it('should return a key pair in jwk format to stdout', async function () {
+      const kid = 'avi';
+      const { exitCode, stdout } = await executeCli(['generate-key', '-s', '-i', kid]);
+
+      expect(exitCode).toEqual(0);
+
+      const { privateKey, publicKey } = JSON.parse(stdout) as { publicKey: Record<string, unknown>; privateKey: Record<string, unknown> };
+      expect(privateKey).toMatchObject({ kid, alg: 'RSA256' });
+      expect(publicKey).toMatchObject({ kid, alg: 'RSA256' });
+
+      // await expect(parseJwk(privateKey)).resolves.not.toThrow();
+      // await expect(parseJwk(publicKey)).resolves.not.toThrow();
+    });
+
+    it('should return a key pair in jwk format to a file', async function () {
+      const kid = 'avi';
+      const { exitCode, stdout } = await executeCli(['generate-key', '-i', kid, '-p', '/tmp/token-cli-test-workdir']);
```

Comment:
throughout the code, you used hardcoded value for the files

Reply:
not anymore